src/components/careers/Positions/index.tsx

```diff
@@ -19,43 +19,30 @@ const Text = styled.p`
   }
 `
 
-export type Department = {
-  id: number
-  name: string
-  jobs: Job[]
-  parent_id: unknown
-  child_ids: number[]
-}
-
-type JobMeta = {
-  id: number
-  name: string
-  value: unknown
-  value_type: string
-}
-
 export type Job = {
```

Comment:
Do they provide types?

Reply:
Couldn't find any. Also had to look through their embed script to find the URL as it is not documented in their API but their support assured me that it can be used without any rate limits.